src/components/WalletStatementModal/index.js

```diff
@@ -0,0 +1,17 @@
+import {Component} from 'react';
+import {Linking} from 'react-native';
+import {walletStatementPropTypes} from './WalletStatementModalPropTypes';
+
+class WalletStatementModal extends Component {
+    componentDidMount() {
+        Linking.openURL(this.props.statementPageURL);
+    }
+
+    render() {
+        return null;
+    }
+}
```

Comment:
I'm not familiar with how we do Linking but is this how it works? Should we just render the linking inside a div instead of doing it on a componentDidMount?

Reply:
> I believe we'd only want to openURL once and if we put it in the render method it would get called repeatedly.

This is a good concern - I think we can resolve this by using a pureComponent.  It just feels a little odd to create a class component that doesn't render anything

Reply:
Just to be clear, did this need to be just `PureComponent` or did you want to see if we can get rid of the class component completely?

Reply:
Doesn't need to be a purecomponent but would be nice to get rid of a class component that resolves the problem of not rendering this multiple times.  I was looking and PureComponent or maybe React.memo might work here?

Reply:
from 1:1, this needs to not be a link but actually it's own view within the modal